capstone/src/main/webapp/js/graphs.js

```diff
@@ -0,0 +1,61 @@
+const script = document.createElement('script');
+script.src = 'https://www.gstatic.com/charts/loader.js';
+script.defer = true;
+
+window.initTuitionData = function() {
+  google.charts.load('current', {packages: ['corechart', 'bar']});
+  google.charts.setOnLoadCallback(graphGrossNetTuition);
+
+  /**
+   * The callback function that will generate the double
+   * bar graph. Uses Google Charts API and JSON object
+   * consisting of college tuition data.
+   */
+  function graphGrossNetTuition() {
+    const data = deserializeTuitionData();
+    const options = {
+      title: 'Gross and Net Price of Colleges',
+      hAxis: {
+        title: 'Your Colleges',
+      },
+      vAxis: {
+        title: 'Tuition Price (USD)',
+      },
+    };
+    const chart =
+      new google.visualization.ColumnChart(document.getElementById('data'));
+    chart.draw(data, options);
+  }
+
+  /**
+   * Helper function to fetch elements from JSON servlet.
+   *
+   * @return {google.visualization.DataTable} data
+   * the DataTable corresponding to the tuition data
+   */
+  function deserializeTuitionData() {
+    const data = new google.visualization.DataTable();
+    fetch('/tuition-data')
+        .then((response) => response.json())
+        .then((tuitionInfo) => populateDataTable(data, tuitionInfo));
+    return data;
```

Comment:
Why do we return the data? I think fetch('/tuition-data') will execute asynchronously so when we return data the table might have not been initialized yet.